.github/workflows/tests_primary.yml

```diff
@@ -61,25 +61,12 @@ jobs:
     - run: ./utils/upload_flakiness_dashboard.sh ./test-results/report.json
       if: always()
       shell: bash
-    - name: Upload blob report to Azure
-      if: always() && github.event_name == 'push'
-      run: az storage blob upload-batch -s test-results/blob-report -d '$web/run-${{ github.run_id }}-${{ github.run_attempt }}-${{ github.sha }}' --connection-string "${{ secrets.AZURE_CONNECTION_STRING_FOR_BLOB_REPORT }}"
-    - name: Upload blob report to GitHub
-      uses: actions/upload-artifact@v3
-      if: always() && github.event_name == 'pull_request'
-      with:
-        name: blob-report-${{ github.run_attempt }}
-        path: test-results/blob-report
-        retention-days: 30
-    - name: Write the pull request number in an file
-      if: always() && github.event_name == 'pull_request'
-      run: echo '${{ github.event.number }}' > pull_request_number.txt;
-    - name: Upload artifact with pull request number
-      uses: actions/upload-artifact@v3
-      if: always() && github.event_name == 'pull_request'
+    - name: Upload blob report
```

Comment:
I think we should also migrate `PWTEST_BLOB_REPORT` things inside our config, like our users would do.